repo/packages/C/cassandra/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "2.0",
+  "name": "cassandra",
+  "version": "0.2.0-2.2.5",
+  "scm": "https://github.com/mesosphere/cassandra-mesos.git",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Cassandra running on DCOS",
+  "framework": true,
+  "tags": ["data", "database", "nosql"],
+  "preInstallNotes":"You are about to install DCOS Cassandra Service.",
+  "postInstallNotes": "DCOS Cassandra Service is installing.",
```

Comment:
Just a suggestion:
Other alternative notes:
- `DCOS Cassandra Service is getting installed`
- `Installation of DCOS Cassandra Service is in progress`